### ISSUE TITLE
CityScapes Pretrained model

### ISSUE THREAD

**Issue by vineetgarg93** [jfzhang95/pytorch-deeplab-xception#62]
Hi,

Do you have the pre-trained weights for cityscapes dataset with resnet101 backbone.

**Comment by xwasco** [comment]
Hi @jfzhang95 , i am interested too :)

Thanks a lot!

**Comment by MartinHahner88** [comment]
thanks for the code @jfzhang95 

I am also curious if you have some weights after training on Cityscapes.
 
The highest mIoU I could achieve with the resnet backbone was ~67% mIoU on the Cityscapes validation set, but this seems fairly low.

Here are my training details, 
I trained for 80 epochs with the following parameters (after merging your code into my codebase):
```
{
    "seed": 0,
    "n_gpu": 1,
    "visualization": {
        "tensorboardX": true
    },
    "metrics": [
        "pixel_accuracy",
        "pixel_accuracy_class",
        "mean_intersection_over_union",
        "frequency_weighted_intersection_over_union"
    ],
    "trainer": {
        "type": "DeepLabV3Plus_Trainer",
        "args": {
            "iterations": 59520,
            "val_freq": 744,
            "monitor": "val_loss",
            "monitor_mode": "min"
        }
    },
    "architectures": [
        {
            "type": "deeplab_v3_plus",
            "args": {
                "backbone": "resnet",
                "output_stride": 16,
                "num_classes": 19,
                "sync_bn": false,
                "freeze_bn": false,
                "pretrained": true
            },
            "optimizer": {
                "type": "SGD",
                "args": {
                    "lr": 0.01,
                    "momentum": 0.9,
                    "weight_decay": 0.0005,
                    "nesterov": false
                }
            },
            "lr_scheduler": {
                "type": "poly_decay",
                "args": {
                    "power": 0.9
                }
            },
            "loss": "cross_entropy_2d"
        }
    ],
    "data_loader": {
        "type": "CityScapes_Loader",
        "args": {
            "base_size": 513,
            "crop_size": 513,
            "batch_size": 4,
            "num_workers": 4,
            "mean": "0., 0., 0.",
            "std": "1., 1., 1."
        }
    }
}
```

**Comment by plzhai** [comment]
> thanks for the code @jfzhang95
> 
> I am also curious if you have some weights after training on Cityscapes as well.
> 
> The highest mIoU I could achieve with the resnet backbone was ~67% mIoU on the Cityscapes validation set, but this seems fairly low.
> 
> Here are my training details,
> I trained for 80 epochs with the following parameters (after merging your code into my codebase):
> 
> ```
> {
>     "seed": 0,
>     "n_gpu": 1,
>     "visualization": {
>         "tensorboardX": true
>     },
>     "metrics": [
>         "pixel_accuracy",
>         "pixel_accuracy_class",
>         "mean_intersection_over_union",
>         "frequency_weighted_intersection_over_union"
>     ],
>     "trainer": {
>         "type": "DeepLabV3Plus_Trainer",
>         "args": {
>             "iterations": 59520,
>             "val_freq": 744,
>             "monitor": "val_loss",
>             "monitor_mode": "min"
>         }
>     },
>     "architectures": [
>         {
>             "type": "deeplab_v3_plus",
>             "args": {
>                 "backbone": "resnet",
>                 "output_stride": 16,
>                 "num_classes": 19,
>                 "sync_bn": false,
>                 "freeze_bn": false,
>                 "pretrained": true
>             },
>             "optimizer": {
>                 "type": "SGD",
>                 "args": {
>                     "lr": 0.01,
>                     "momentum": 0.9,
>                     "weight_decay": 0.0005,
>                     "nesterov": false
>                 }
>             },
>             "lr_scheduler": {
>                 "type": "poly_decay",
>                 "args": {
>                     "power": 0.9
>                 }
>             },
>             "loss": "cross_entropy_2d"
>         }
>     ],
>     "data_loader": {
>         "type": "CityScapes_Loader",
>         "args": {
>             "base_size": 513,
>             "crop_size": 513,
>             "batch_size": 4,
>             "num_workers": 4,
>             "mean": "0., 0., 0.",
>             "std": "1., 1., 1."
>         }
>     }
> }
> ```

can you share the pretrained model on cityscapes?
